ViewTemplates/About/default_license.blade.php

```diff
@@ -23,7 +23,7 @@
 
 <div class="d-flex flex-column flex-md-row gap-2">
     <div>
-        <img src="@media('images/agpl_logo.svg', false, $this->container->application)" class="img-fluid">
+        <img src="@media('images/agpl_logo.svg', false, $this->container->application)" class="img-fluid" alt="GNU Affero General Public License">
```

Comment:
I believe this should read "Logotype of the GNU Affero General Public License version 3" as per [the MDN explanation](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt):

> **Think of it like this:** When choosing `alt` strings for your images, imagine what you would say when reading the page to someone over the phone without mentioning that there's an image on the page.

If I said to someone over the phone "GNU AGPL" they might misunderstand it as me summarizing a long text area with the license text. Or, at least, that's what I would understand if that was how it was conveyed to me over the phone like that.